modules/architecture-rhcos-updating-bootloader.adoc

```diff
@@ -0,0 +1,88 @@
+// Module included in the following assemblies:
+//
+// * architecture/architecture_rhcos.adoc
+
+[id="architecture-rhcos-updating-bootloader.adoc_{context}"]
+= Updating the bootloader using bootupd
+
+To update the bootloader using bootupd, you must either install bootupd on {op-system} machines manually, or provide a machine config with the enabled systemd unit. Unlike grubby or other bootloader tools, bootupd does not manage kernel space configuration such as passing kernel arguments.
```

Comment:
```suggestion
To update the bootloader by using `bootupd`, you must either install `bootupd` on {op-system} machines manually or provide a machine config with the enabled `systemd` unit. Unlike grubby or other bootloader tools, `bootupd` does not manage kernel space configuration such as passing kernel arguments.
```

Reply:
Applied, thanks - and good to be in sync with the Release Notes that now use the same conventions thanks to your previous review! :)